runtime/runc_jailer.go

```diff
@@ -183,7 +188,13 @@ func (j *runcJailer) BuildJailedRootHandler(cfg *Config, socketPath *string, vmI
 						return errors.Wrapf(err, "failed to stat drive %q", drivePath)
 					}
 
-					if err := copyFile(drivePath, newDrivePath, info.Mode()); err != nil {
+					// If the file will be mounted as read-only, drop write permissions
+					mode := info.Mode()
+					if *d.IsReadOnly {
+						mode &= 0555
```

Comment:
Can this just be `0400`? Off the top of my head I can't think of why group+other would need any access and I can't think of why user would need executable access.

Reply:
Also, in the case where this isn't read only, can we set to `0600`?

Reply:
Yeah. 0400 / 0600 is much simpler.